build.cake

```diff
@@ -153,6 +153,30 @@ Task("Zip-Files")
     Zip(binDir, packagePath, files);
 });
 
+Task("Create-Chocolatey-Packages")
+    .IsDependentOn("Copy-Files")
+    .WithCriteria(() => isRunningOnWindows)
+    .Does(() =>
+{
+  // Create Cake package.
+  ChocolateyPack("./nuspec/Cake.Portable.nuspec", new ChocolateyPackSettings {
+      Version = semVersion,
+      Files = new [] {
+        new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/Cake.exe")},
```

Comment:
@patriksvensson based on your suggestion, I have done this:

![image](https://cloud.githubusercontent.com/assets/1271146/11047593/4c968472-872b-11e5-9dd0-5395ab2b93bd.png)

As making the directory absolute when it is declared causes other issues.  But when I try to run the build, I get this:

![image](https://cloud.githubusercontent.com/assets/1271146/11047616/69f9b7dc-872b-11e5-852b-27adf92cb836.png)

Should a DirectoryPath + File put the / in for me?